skaffold.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: skaffold/v2beta6
+kind: Config
+metadata:
+  name: open-aea
+build:
+  tagPolicy:
+    envTemplate:
+      template: "1.10.0"
```

Comment:
This file should be referenced in the bump aea version script, so it also bumps the version here!